client/components/QuestionsAndAnswers/QAndAComponents/QAndA.jsx

```diff
@@ -1,33 +1,68 @@
 import React from 'react';
-import Grid from '@material-ui/core/Grid';
-import TextField from '@material-ui/core/TextField';
-import Button from '@material-ui/core/Button';
-import SearchIcon from '@material-ui/icons/Search';
-import InputBase from '@material-ui/core/InputBase';
-import GridList from '@material-ui/core/GridList';
-import GridListTile from '@material-ui/core/GridListTile';
-import IconButton from "@material-ui/core/IconButton";
-import InputAdornment from "@material-ui/core/InputAdornment";
+import { Grid, Typography, makeStyles } from '@material-ui/core';
+import Helpful from './Helpful.jsx';
+var dateFormat = require('dateformat');
 
+const useStyles = makeStyles((theme) => ({
+  underlined: {
+    textDecoration: 'underline'
+  },
+  textSpacing: {
+    marginLeft: '10px',
+    marginRight: '10px'
+  },
+  verticalSpace: {
+    marginTop: '15px',
+    marginBottom: '15px'
+  }
+}));
 
 const QAndA = (props) => {
+  const classes = useStyles();
   return (
-    <div>
-      {/* Iterate through results
-            find the 2 questions with most helpfulness
-            find the most helpful answer related to that
-            if they contain photos, display those photos
-      */}
-      <div>
-        <h4><em>Q: {props.questions.results[1].question_body}</em></h4>
-        <h4><em>A: </em>{props.questions.results[1].answers[1628393].body}</h4>
-      </div>
-      <div>
-        <h4><em>Q: {props.questions.results[2].question_body}</em></h4>
-        <h4><b>A: </b>{props.questions.results[2].answers[1628412].body}</h4>
-      </div>
-    </div>
+    <Grid container spacing={2}>
+      <Grid className={classes.verticalSpace}></Grid>
+      <Grid container direction="row" spacing={2}>
+        <Grid item xs={8}>
+          <Typography variant="h6"><b>Q: {props.question.question_body}</b></Typography>
+        </Grid>
+        <Grid container justify="flex-end" xs={4}>
+          <Grid item className={classes.textSpacing}>
+            <Helpful question={props.question}/>
+          </Grid>
+          <Grid item>
+            <Typography variant="caption">|</Typography>
+          </Grid>
+          <Grid item className={classes.textSpacing}>
+            <Typography variant="caption" className={classes.underlined}>Add Answer</Typography>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid container spacing={2} xs={12}>
+        <Grid item>
+          {/* <Typography variant="h6"><b>A: </b>{props.question.answers[1628393].body}</Typography> */}
```

Comment:
Do you still need this line? Seems like a line no longer used and can be deleted.

Reply:
I changed this to be a placeholder, which says "Future answer goes here."  Good catch, Julia :)